cool-core/src/main/java/com/nus/cool/core/cohort/refactor/CohortResultLayout.java

```diff
@@ -0,0 +1,41 @@
+package com.nus.cool.core.cohort.refactor;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.jsonschema.JsonSerializableSchema;
+import java.util.ArrayList;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+/**
+ * Layout class to facilitate json serialization of cohort query.
+ */
+@Data
+@JsonSerializableSchema
+public class CohortResultLayout {
```

Comment:
So is this class expected to store a meta file describing cohorts in the folder?

Reply:
This will store an original query and corresponding cohort meta-result in the following format:
`{"query": original_query, "query_res": [ {"name": "fileName", "cohortName": cohortName, "cohortSize": cohortSize} ]}`
This is to facilitate the future use of frontend, according to @KimballCai